backtracking/all_subsequences.py

```diff
@@ -1,35 +1,53 @@
 """
-In this problem, we want to determine all possible subsequences
```

Comment:
Import block is un-sorted or un-formatted
   |
 8 |   """
 9 |   
10 | / from __future__ import annotations
11 | | from typing import Any
12 | | 
13 | | 
14 | | def generate_all_subsequences(sequence: list[Any]) -> None:
   | |_^ I001
15 |       """Kickstarts the backtracking algorithm.
   |
   = help: Organize imports